src/components/Welcome/Welcome.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Welcome.css';
 
-// Create a Welcome function returning the HTML of welcome block
+function Welcome() {
+  return (
+    <section className="welcome">
+      <span className="welcome__text">Sticky Header!</span>
+    </section>
+  );
+}
```

Comment:
better to have functional components as an arrow functions, text should be moved to a new line
```suggestion
const Welcome = () => (
    <section className="welcome">
      <span className="welcome__text">
        Sticky Header!
      </span>
    </section>
);
```